analyzers/tests/SonarAnalyzer.UnitTest/Extensions/SyntaxNodeExtensionsTest.cs

```diff
@@ -1013,7 +1061,7 @@ public void ParameterList_PrimaryConstructors(string type)
             var node = NodeBetweenMarkers($$"""
                 $$public {{type}} C(int p)
                 {
-                    
+
```

Comment:
What is this notation?
`$$int i;$$`

Reply:
See `NodeBetweenMarkers`. `$$` is used as a (somewhat confusing) UT scaffolding annotation